### ISSUE TITLE
Human-readable output

### ISSUE THREAD

**Issue by segeljakt** [laumann/compiletest-rs#234]
By default compiletest seems to dump the output in json which is good but difficult to read. Human readable output can be added manually by including a compiler directive at the top of the file.

```rust
// compile-flags: --error-format=human
fn foo() {

}
```

Would it be possible to add an option to `Config` so that this does not need to be done for every file?

**Comment by Munksgaard** [comment]
Good idea! Pull requests are certainly welcome, but I cannot promise that anyone else is going to come around to adding it anytime soon.
